### PR TITLE
Main net support

### PR DESCRIPTION
**Pull request by yaroslavr-itd**
Moved all hard coded variables to environment variables.

New environment variables, that need to be added to run funder package.
PAKET_FUNDER_SEED - seed of account, that will be used for funding accounts of users.
PAKET_XPUB - xpub key for generating payment addresses.
PAKET_MINIMUM_MONTHLY_ALLOWANCE - minimum amount of euro cents, for handling per month.
PAKET_BASIC_MONTHLY_ALLOWANCE - amount of euro cents, that user can purchase after passing basic kyc test.
PAKET_EURO_PRICE_IN_BUL - amount in BUL that need to be payed for 1 euro cent.
PAKET_ETHERSCAN_API_KEY - api key for accessing etherscan.io web API.

### CODE REVIEW

**Comment by israellevin** [comment]
There was some talk of replacing etherscan with something that does not require an API key because of some limitations they pose. Is this something you are still investigating?

File: db.py [hunk 1]
```diff
@@ -10,16 +10,16 @@
 import csl_reader
 
 LOGGER = logging.getLogger('pkt.funder.db')
-SEED = ('client ancient calm uncover opinion coil priority misery empty favorite moment myth')
-XPUB = 'tpubD6NzVbkrYhZ4XMSG7EWChwJXwfByid9TdZRVaej1rpDTHV3WamyuApceF5DDZXetx8kbH82NouoazYqPeCEZWWeXHZ1do5LBCe5xMcZYeGe'
+DEBUG = bool(os.environ.get('PAKET_DEBUG'))
```
Comment (israellevin) [inline db.py]: In the context of this module, this only indicates that we are using testnet, right?

If so, I'm considering changing the name of the constant - something like:
```python
TESTNET = bool(os.environ.get('PAKET_DEBUG'))```

Not sure about it.

**Comment by israellevin** [comment]
@yaroslavr-itd , let's compile a clean and full list of only the variables that need to be switched to move from test to production. I'll start:
1. PAKET_DEBUG
2. PAKET_USER_ISSUER
3. PAKET_FUNDER_SEED
4. PAKET_PAYMENT_XPUB

Currently, we will also need to change PAKET_HORIZON_SERVER, but I think we should just give it sane defaults according to PAKET_DEBUG.

File: db.py (hunk 1 above)
Comment (yaroslavr-itd) [reply db.py]: No, in context of this module it is also used to indicate debug mode, to prevent using debug-only function - db.get_users()

**Comment by yaroslavr-itd** [comment]
@israellevin I've tried to found free ethereum explorer with web API, but all of them have some limitation, like limit of requests per second or demand to mention their service on our website. As option, we can use web3py to obtain ethereum balances. What is your opinion?

**Comment by israellevin** [comment]
@yaroslavr-itd , what would be the overhead of using web3 to get balances? In terms of storage space for the blockchain, bandwidth, and reliability? Can we do this in some sort of SPV mode, like a mobile phone client?

**Comment by israellevin** [comment]
@yaroslavr-itd , what would be the overhead of using web3 to get balances? In terms of storage space for the blockchain, bandwidth, and reliability? Can we do this in some sort of SPV mode, like a mobile phone client?

File: db.py (hunk 1 above)
Comment (israellevin) [reply db.py]: :+1:

**Pull request merged**